### ISSUE TITLE
[BUG] FSMP 2.5.1 not loading on SE 1.5.97 SKSE 2.0.20

### ISSUE THREAD

**Issue by FuzzyWuggzy** [DaymareOn/hdtSMP64#156]
**Describe the bug**
After installing FSMP using Vortex there's still no functioning physics, the MCM menu registers so I tried resetting SMP in it but that just causes `Script command "smp" not found.` to appear in the console multiple times.

**Desktop (please complete the following information):**
 - OS: WINE (Windows 10 Prefix)

** FSMP**
- 2.5.1
- CUDA: No_CUDA
- AVX: AVX

**SKSE**
- 2.0.20

**Skyrim**
- 1.5.97

**Required files**
- [hdtSMP64.log](https://github.com/user-attachments/files/17985351/hdtSMP64.log)
- Can't find a config.xml

**Additional context**
Inside the SKSE log I found the line ```plugin [PATH TO SE INSTALL]\Data\SKSE\Plugins\\hdtSMP64.dll (00000001 hdtSMP64 00030F34) reported as incompatible during query (handle 7)```, so SKSE doesn't seem to be loading the dll.

Thank you

**Comment by snottyone** [comment]
it's configs.xml, and that is in the FSMP mod folder here `\SKSE\Plugins\hdtSkinnedMeshConfigs\configs.xml`

check if you have all of
- **PapyrusUtil SE** [Nexus Link](<https://www.nexusmods.com/skyrimspecialedition/mods/13048>)
- **JContainers SE** [Nexus Link](<https://www.nexusmods.com/skyrimspecialedition/mods/16495>)
- **ConsoleUtilSSE NG** [Nexus Link](<https://www.nexusmods.com/skyrimspecialedition/mods/76649>)

(note that consoleutil needs to be the NG version)
check that none of those gets overwritten (if so, set to 'load after')

if you started your game with the updated FSMP, and had one of those mods above missing, reinstalling FSMP should fix your problem.

otherwise, please attach skse64.log and configs.xml

**Comment by FuzzyWuggzy** [comment]
My bad, no wonder I couldn't find it.

I already have all the requirements listed on the Nexus mods page installed, including the ones you listed, so I don't think that's the issue.

Here's the files requested:

Configs.xml (changed extension so I could upload): [configs.txt](https://github.com/user-attachments/files/17998192/configs.txt)
SKSE64 log: [skse64.log](https://github.com/user-attachments/files/17998394/skse64.log)

Thank you for replying.

**Comment by snottyone** [comment]
hmm, no obvious problem preventing FSMP from working in those two files
(incidentals: [Simple Lock-On SE](https://www.nexusmods.com/skyrimspecialedition/mods/18086) also doesn't load, works only with 1.5.39, uninstall, and you have both Trainwreck and Crashlogger installed, uninstall Trainwreck)

one thing to try - start reinstalling FSMP, click Next once, check if the installer correctly detects your game version (cancel installation after checking that)

**Comment by FuzzyWuggzy** [comment]
Thanks for the spot, I installed True Directional Movement when I realised that Simple Lock-On didn't work and forgot to uninstall the latter. I've also uninstalled Trainwreck as you recommended.

I made sure when I installed it before, Vortex correctly identifies my version of Skyrim.

![Screenshot_20241203_223641](https://github.com/user-attachments/assets/31aa654f-ba3a-45c2-a534-a7a27b2fbe83)

**Comment by FuzzyWuggzy** [comment]
Update: I just created a new WINE prefix, installed Skyrim SE using the same source files as the last one and only installed FSMP, its requirements and KS hairdos SMP and physics are working perfectly, so there's something wrong with my original setup.

As I started modding the game manually before switching to Vortex (as it never used to work before) I think I'll just start afresh with the new prefix for the sake of my sanity.

I'll close this now as it was my fault. Thank you for your help @snottyone
